tc_aws/loaders/__init__.py

```diff
@@ -1 +1,48 @@
-# coding: utf-8
\ No newline at end of file
+# coding: utf-8
+
+import urllib2
+
+def _get_bucket_and_key(context, url):
+    url = urllib2.unquote(url)
+
+    bucket = context.config.get('S3_LOADER_BUCKET', default=None)
+
+    if not bucket:
+        bucket, key = _get_bucket(url, root_path=context.config.S3_LOADER_ROOT_PATH)
+    else:
+        key = url
+
+    return bucket, key
+
+
+def _get_bucket(url, root_path=None):
+    """
+    Returns a tuple containing bucket name and bucket path.
+    url: A string of the format /bucket.name/file/path/in/bucket
```

Comment:
My bad, just saw it was a moved file, we'll re-work the documentation another time ;)